gateway-server/src/main/java/org/apache/knox/gateway/services/token/impl/DefaultTokenStateService.java

```diff
@@ -51,13 +51,16 @@
 
   /* token eviction interval in seconds, default is 5 mins */
   private long tokenEvictionInterval;
+  /* grace period (in seconds) after which an expired token should be evicted, default 5 mins */
```

Comment:
nit: This comment won't change when the default value changes in the future. I try not to include values in comments